src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlConnectionPoolGroupProviderInfo.cs

```diff
@@ -2,26 +2,27 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using Microsoft.Data.Common;
 using Microsoft.Data.ProviderBase;
 
 namespace Microsoft.Data.SqlClient
 {
     sealed internal class SqlConnectionPoolGroupProviderInfo : DbConnectionPoolGroupProviderInfo
     {
         private string _alias;
-        private System.Security.PermissionSet _failoverPermissionSet;
         private string _failoverPartner;
         private bool _useFailoverPartner;
+#if NETFRAMEWORK
+        private System.Security.PermissionSet _failoverPermissionSet;
```

Comment:
Could you remove this namespace and use the `using` directive?
```suggestion
        private PermissionSet _failoverPermissionSet;
```